civiform/civiform
# Add custom confirmation message to the "Application Submitted" page

### Description

Add custom confirmation message to the North Star "Application Submitted" page.

## Release notes

n/a

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. Enable North Star
2. Find a program with a custom confirmation message (or create one. See screenshot in #7907)
3. As an applicant, apply to that program
4. Observe the "Application Submitted" page with the custom confirmation message

### Issue(s) this completes

Fixes issue #7907
